src/pages/events/tutorial/events-action-code-debugging.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Extend Adobe Commerce with Events and App Builder
```

Comment:
Global in all .md files: 

* Titles are in sentence case.
* Titles are not repeated
* Add `Events` as a keyword in all topics
* Remove the `noIndex` line
* Remove blank lines within the frontmatter (the text between the `---` lines)
* Ensure that the first heading in each topic is a head1 and generally matches the title defined in the frontmatter. Adjust subsequent headings accordingly.
* All headings are in sentence case